applications/CompressiblePotentialFlowApplication/tests/cpp_tests/test_compressible_perturbation_potential_element.cpp

```diff
@@ -67,40 +146,63 @@ KRATOS_TEST_CASE_IN_SUITE(CompressiblePerturbationPotentialFlowElementRHS, Compr
     GenerateCompressiblePerturbationElement(model_part);
     Element::Pointer pElement = model_part.pGetElement(1);
 
-    AssignPotentialsToNormalCompressiblePerturbationElement(pElement);
+    std::array<double, 3> potential{1.0, 20.0, 50.0};
+    AssignPotentialsToNormalCompressiblePerturbationElement(pElement, potential);
 
     // Compute RHS
     Vector RHS = ZeroVector(3);
 
     const ProcessInfo& r_current_process_info = model_part.GetProcessInfo();
     pElement->CalculateRightHandSide(RHS, r_current_process_info);
 
-    std::vector<double> reference{146.2643261263345,-122.1426284341492,-24.12169769218525};
+    std::vector<double> reference{131.4361747323354,-113.768439084114,-17.66773564822145};
```

Comment:
```suggestion
    std::array<double, 3> reference{131.4361747323354,-113.768439084114,-17.66773564822145};
```

Reply:
> minor, mostly accessing the array3 with reference (I might have missed some), if they are anyway const

Thanks @philbucher, I'll go through your suggestions

Reply:
This doesn't compile:

kratos/includes/exception.h:87:16: error: no match for ‘operator<<’ (operand types are ‘std::stringstream {aka std::__cxx11::basic_stringstream<char>}’ and ‘const std::array<double, 3ul>’)
         buffer << rValue;